ruby_audit.gemspec

```diff
@@ -7,6 +7,7 @@ Gem::Specification.new do |spec|
   spec.version       = RubyAudit::VERSION
   spec.authors       = ['Jeff Cousens, Mike Saelim', 'John Zhang', 'Cristina Muñoz']
   spec.email         = ['opensource@civisanalytics.com']
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
Rubocop encourages everyone to require MFA for rubygems operations like publishing gems.

https://guides.rubygems.org/mfa-requirement-opt-in/

This will take effect after this gem is published to rubygems, with this line of metadata in.  We already have MFA set up for Civis's account, right?